### ISSUE TITLE
Version 0.5.0 is not available on NPM

### ISSUE THREAD

**Issue by NiklasMerz** [johanblomgren/cordova-plugin-indexappcontent#26]
I would be nice to install version 0.5.0 from NPM instead of Github.

**Comment by MarcoEidinger** [comment]
Hi Niklas,

do you still require it? I saw that https://www.npmjs.com/package/@ionic-native/index-app-content exists.

Only @johanblomgren can publish in NPM. Maybe for version 0.6.0 (next week), it would make sense?!?

**Comment by NiklasMerz** [comment]
Yes publishing the next version on NPM would be nice. I built the ionic native wrapper. This is not the plugin code, just some typescript interfaces for this plugin.
